server/src/api/api.graphql

```diff
@@ -78,3 +84,54 @@ type Category {
     # The human-readable, user-facing name for this category
     category_name: String!
 }
+
+enum RequestStatus {
+	SUBMITTED
+	APPROVED
+	DENIED
+	ABANDONED
+    CANCELLED
+	READY_FOR_PICKUP
+	FULFILLED
+	RETURNED
+	LOST
+	DAMAGED
+}
+
+type Request {
```

Comment:
So what happened to a shopping cart/inventory idea? With an inventory ui, items grouped by status, it could be a frontend concept. When an item status changes, like new qty requested it can merge into whatever exists - update status accounts for this. Can a request delete itself if it goes to qty 0 (idk, make sure it doesn't linger in the system). 
- we should have websockets on the admin side in case requests change and you approve a changed request

Reply:
It's not a shopping cart perse because of previously discussed complications surrounding handling that, like having a timer to reserve your items and what not.  Merging requests is potentially a good idea, but seems like something I should just make an issue for rn.